node-module-system/add.js

```diff
@@ -0,0 +1,3 @@
+const sum = (num1, num2) => num1 + num2;
```

Comment:
You can use arrow syntax like if you'd like, it's a fairly common practice. However, my personal preference is to use arrow function syntax _only_ when I'm passing a function as a parameter. I prefer to see the word `function` on top-level functions because I think it makes the code easier to read and understand.

Another side benefit is that you can say `export default function sum()`, which you can't do with arrow functions. (`export default const sum = () => {}` is a syntax error for reasons I haven't been able to discover)